implementation.js

```diff
@@ -0,0 +1,27 @@
+/*! https://mths.be/repeat v1.0.0 by @mathias */
+
+'use strict';
+
+var RequireObjectCoercible = require('es-abstract/2019/RequireObjectCoercible');
+var ToString = require('es-abstract/2019/ToString');
+var ToInteger = require('es-abstract/2019/ToInteger');
+
+module.exports = function repeat(count) {
+    var O = RequireObjectCoercible(this);
```

Comment:
```suggestion
	var O = RequireObjectCoercible(this);
```
there's a lot of spaces in this repo still :-)

Reply:
Whoops 🙃
I'm not using my usual editor and I still have to properly configure this one 😂